test/integration/narrow/routes/endemics/date-of-visit-ms.test.js

```diff
@@ -255,17 +255,60 @@ describe('POST /claim/endemics/date-of-visit handler', () => {
 
     const $ = cheerio.load(res.payload)
     expect(res.statusCode).toBe(400)
-    expect($('p.govuk-error-message').text().trim()).toEqual(
-      'Error: The date of review must be a real date'
-    )
+    expect($('#main-content > div > div > div > div > div > ul > li > a').text().trim()).toEqual('Enter a date in the boxes below')
```

Comment:
Can we not give the anchor an ID, and use that to CSS select it here. This seems quite brittle if the layout ever changes very slightly.

Reply:
I'd go one further if its ok with you and @mattyod - can we start using snapshots of the HTML? It would save us faffing around to check the content.

Reply:
I'm very much **not** a fan of snapshot testing for HTML, it's almost impossible to parse a several hundred line html file when testing or reviewing and they tend to need updating frequently. So it ends up just becoming something the people run `jest -u` to fix and then they move on.

We are in the process of moving towards using [Testing Library](https://testing-library.com/) which will provide a much more robust way of checking something like this, so I'd rather focus on that than spend time updating these tests to use snapshots.

Testing Library example for that selector:
```
getByRole(document.body, 'link', { name: 'Enter a date in the boxes below' })
```

Reply:
Yeah very fair points! Ive removed them for now and updated the xpath we use to get the error message. We cant easily add a data-testid because its a govuk component.